src/modules/utils/simpleshell/SimpleShell.cpp

```diff
@@ -283,7 +283,10 @@ void SimpleShell::on_console_line_received( void *argument )
 
         } else if (cmd == "fire") {
             // these are handled by Laser module
-
+			
+		} else if (cmd == "sendSPIdrv") {
+			// these are handled by MotorDriverControl module
+			// This isn't the best way to do this. Event handler could pass a flag to see if anyone handled the event, then, if not, call a default event handler; or register the cmd_str or hash with the cmdline event dispatcher, etc...Other ways to work this out.
```

Comment:
I am well aware that there are better ways, however the historical structure of the code does not allow events to return values, so this was the easiest way to accomplish the task. It would be too disruptive to change the code at this time. Therefore you may remove this comment.

Reply:
Removed the gratuitous comment.  (Sorry, I know you are well aware.)